.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I guess you committed this file by mistake :)

Reply:
Yep, but it might be handy to have it, or add .vscode to the gitignore.